README.rst

```diff
@@ -234,4 +234,10 @@ Releasing
 2. Publish a new release and tag ``vX.Y.Z``, using autogenerated changelog
 3. Watch for deployment notifications in the Mozilla ``#kinto-standup`` Slack channel.
 
+In order to deploy manually (eg. to production):
+
+1. Go to [deployment workflow page](https://github.com/mozilla-sre-deploy/deploy-remote-settings/actions/workflows/gcp.yaml)
+2. Click on `Run workflow`
+3. Pick `Branch=main`, `Environment=prod`, `ref=vX.Y.Z`, and click `Run workflow`
```

Comment:
Worth throwing a screenshot in here?

Reply:
In these instructions the most important information is the link to the workflow, the UI to run it is rather obvious. I'd say this is enough :)